src/include/distributed/adaptive_executor.h

```diff
@@ -1,6 +1,10 @@
 #ifndef ADAPTIVE_EXECUTOR_H
 #define ADAPTIVE_EXECUTOR_H
 
+#include "postgres.h"
+#include "funcapi.h"
```

Comment:
nothing should be included in the same block as `postgres.h` a lot of include files from postgres assume this header has already been included.

Once we add rules to uncrustify to reorder includes alphabetically this will break.
Instead it is better to include `funcapi.h` in the block below where we include both our and postgres' header files.